src/client/tablet_client.cc

```diff
@@ -523,12 +521,13 @@ bool TabletClient::DeleteOPTask(const std::vector<uint64_t>& op_id_vec) {
     return true;
 }
 
-bool TabletClient::GetTermPair(uint32_t tid, uint32_t pid, uint64_t& term, uint64_t& offset, bool& has_table,
-                               bool& is_leader) {
+bool TabletClient::GetTermPair(uint32_t tid, uint32_t pid, ::openmldb::common::StorageMode storage_mode, uint64_t& term,
+                               uint64_t& offset, bool& has_table, bool& is_leader) {
     ::openmldb::api::GetTermPairRequest request;
     ::openmldb::api::GetTermPairResponse response;
     request.set_tid(tid);
     request.set_pid(pid);
+    request.set_storage_mode(storage_mode);
```

Comment:
no need the `storage_mode`. can get the storage_mode from `Table.GetStorageMode()` in tablet.

Reply:
Four functions use storage_mode now. `SendData`, `GetManifest`, `GetTermPair` and `DeleteBinlog`. 
`SendData` is used in `SendIndexData` now and is only available for Memtable.
The other three can use even when the table does not exist.